components/edit-collective/sections/UserSecurity.js

```diff
@@ -60,6 +60,13 @@ class UserSecurity extends React.Component {
     this.setPassword = this.setPassword.bind(this);
   }
 
+  componentDidUpdate() {
+    if (window.location.hash) {
+      const section = document.querySelector(window.location.hash);
+      section.scrollIntoView();
+    }
+  }
```

Comment:
Doing that in `componentDidUpdate` is dangerous; if you start typing in an input somewhere else on the page it will bring you back to this section for each keystroke.